packages/core/src/bundle.ts

```diff
@@ -305,7 +305,6 @@ export function mapTypeToComponent(typeName: string, version: SpecMajorVersion)
       }
     case SpecMajorVersion.Arazzo:
       switch (typeName) {
-        case 'Root.x-parameters_items':
```

Comment:
Do we still use these autogenerated names? I was under impression we've rewritten everything into the NodeType tree.

Reply:
Not a scope of this PR.
Appears it exists on some places in code  - found this mention here as was removing.

Reply:
Please leave a TODO to not forget to replace that later.